types/mdx/package.json

```diff
@@ -10,7 +10,7 @@
     "tsconfigs": ["tsconfig.preact.json", "tsconfig.react.json"],
     "devDependencies": {
         "@types/mdx": "workspace:.",
-        "@types/react": "*",
+        "@types/react": "^18",
```

Comment:
React 16 and 17 are also supported by MDX https://github.com/mdx-js/mdx/blob/6c0f62d1665037a273f19604adfc6c1b8a308084/packages/react/package.json#L46

Reply:
It’s just a dev dependency. 18 is fine.